CHANGES.rst

```diff
@@ -1,3 +1,12 @@
+1.13.0 (unreleased)
+===================
+
+Enhancements
+------------
+* Adds support for legacy predictions (api/v1) and a new parameter for specifying api version (--api_version)
```

Comment:
Can you say "Returns support"?

Also, if you're not going to specify which options are available for `--api_version` here, we should tell them to look at the help page for more information.

Reply:
We also didn't mention that we will try to infer the api_version based on their endpoint url